conference.js

```diff
@@ -1288,6 +1288,24 @@ export default {
             APP.UI.updateUserRole(user);
 
             updateRemoteThumbnailsVisibility();
+
+            const currentRemoteUserCount = room.getParticipantCount() - 1;
+            if (currentRemoteUserCount === 1) {
+                const userId = user.getId();
+
+                window.setTimeout(() => {
+                    // Only update if there is one remote user, no one is
+                    // pinned, self is on large, and the remote user is still
+                    // present.
+                    if ((room.getParticipantCount() - 1 === 1)
+                        && !APP.UI.getPinnedId()
+                        && (APP.UI.getLargeVideoID()
+                             === APP.conference.getMyUserId())
+                        && room.getParticipantById(userId)) {
+                        APP.UI.updateLargeVideo(userId);
```

Comment:
We also have a very similar check in VideoLayout.remoteVideoActive about isPinned, etc, so we may want to have it in a func that would be called either from the timeout or when the stream is received.